src/CodeFormatter/CodeFormatter.csproj

```diff
@@ -1,35 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
```

Comment:
This would actually be `..\..\build.props` for all the projects we have right now.  I basically just did this so I could put the build.props at the top-level without having to specify exactly where the top-level is (in case things move it's one less thing to change for each project).  This is what we do in the corefx repo to find `dirs.props` and `dirs.targets`, but since the build here is not as complex this is probably not necessary.   I'll change this to `..\..\build.props`.